web/src/test/java/se/inera/intyg/webcert/web/service/access/DbDraftAccessTest.java

```diff
@@ -152,9 +152,13 @@ protected void assertAllowToEditUtkastNotLatestMajorVersion(AccessResult actualV
         assertEquals(AccessResultCode.NO_PROBLEM, actualValue.getCode());
     }
 
+    /**
```

Comment:
In this case it may be a bit obvious. However my experience is that there should always be javadoc to describe the intention of the method. (Tests may be excluded.) Method tends to grow over time and get harder and harder to read without knowledge of the of why functionality was added.

Reply:
Feel free to keep it if you like. The assert in itself is pretty clear what it does and what is expected. My experience is that the burden of maintaining javadoc, that runs the risk to not being up to date with the code, is not worth it. The code will be always be the truth. 

If we were writing a library for others to use I would think different about the value.

Reply:
We could maybe discuss this a on our "utvecklarmöte".